crates/composer/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "composer"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bincode = "1.3.3"
+color-eyre = "0.6.2"
+eyre = "0.6.8"
+rodio = { version = "0.17.1", features = ["symphonia-wav"] }
+serde = { version = "1.0.160", features = ["serde_derive"] }
```

Comment:
In tonari codebase, we typically do:
```
bincode = "1"
color-eyre = "0.6"
eyre = "0.6"
```

Only specify the major version past 1.0, and specify minor version before 1.0. This will give cargo the most flexibility assuming that our dependent crates follow the semvar compatibility requirements strictly. @strohel should we do that here as well?

Reply:
@skywhale yeah that's basically what I intended to say (it convoluted way) in my above comment.

The only exception/caveat is that patch (x.y.Z+1) releases *can* introduce new features (if they're backwards compatible), and we can depend on the new features. So _sometimes_ specifying all 3 components for dependencies is needed/correct.

Reply:
I was lazy here and just rolled with whatever `cargo add ...` produced.

Trying to not block this PR further, in 80d20295fb4ed91607e547540320008e50600481 I changed the versions to what @skywhale suggested.

I'll try to get to the philosophical aspect later. Now, I need to go out with the dog. .)

Reply:
Getting back to the caveat @strohel mentions: I think that there is little reason to aim for the correct (i.e. minimal functional lower bound) version range from the get go. We'd have to essentially enumerate all the "features" we use and backtrack them to the minor version they were all included in. That'd be hard.

Also remember, that there's cargo.lock. If someone adds a `dependency = "1.5"` and makes our program work but in fact relies on a feature from `1.5.5` then the exact version pinned in cargo.lock must be `1.5.5+`. I.e. the caveat can only arise when cargo.lock changes which isn't as often and it should be fairly easy to spot what happened in that case. At least as long as the version mismatch manifests visibly - which is something I'd expect in a language typed as strongly as rust.

Reply:
@goodhoko agreed. Perhaps libraries (which do not ship `Cargo.lock`) that are popular should do this, but indeed that seems like an overkill for us.